docs/development/extensions-core/druid-kerberos.md

```diff
@@ -72,7 +72,7 @@ If required, multiple rules can be be joined by newline character and specified
 ### Increasing HTTP Header size for large SPNEGO negotiate header
 In Active Directory environment, SPNEGO token in the Authorization header includes PAC (Privilege Access Certificate) information,
 which includes all security groups for the user. In some cases when the user belongs to many security groups the header to grow beyond what druid can handle by default.
-In such cases, max request header size that druid can handle can be increased by setting `druid.server.http.maxRequestHeaderSize` (default 8Kb) and `druid.router.http.maxRequestBufferSize` (default 8Kb).
```

Comment:
According to the code, its default value is `8KiB`

https://github.com/apache/druid/blob/0c5d1c9725303111df69198a232166ddf00672d2/server/src/main/java/org/apache/druid/server/initialization/ServerConfig.java#L122